getting-started/README.md

```diff
@@ -4,22 +4,143 @@ Amber is designed to make common web-development tasks simple, and easy, maintai
+
+After you create the blog application, switch to its folder:
+
+```bash
+$ cd blog
+```
+
+You can then install your dependencies with this command
+
+```bash
+$ shards install
+```
+
+The `blog` directory has a number of auto-generated files and folders that make up the structure of an Amber application. Most of the work in this tutorial will happen in the `src` folder, but here's a basic rundown on the function of each of the files and folders that Rails created by default:
+
+| File Folder | Purpose |
+| :--- | :--- |
+| config/ | Contains configuration code for databases, environments, routing, and deployment |
+| db/ | Contains SQLite database and database migrations |
+| lib/ | Contains local installation of all dependent shards |
+| public/ | Location for html, css, and javscripts. This will be the default handler for static assets. |
+| spec/ | Location for application specification and tests |
+| src/ | Source code that composes your application |
+| .amber\_secret\_key | This file is used to encrypt application secrets so they are not stored in source control |
+| .gitignore | This file tells git which files and patterns it should ignore. |
+| docker-compose.yml | A file which defines services, networking, and drives for docker deployment. |
+| Dockerfile | A file which contains all the commands needed to build a docker image. |
+| README.md | A brief instruction manual about your application |
+
+## Hello, Amber!
+
+To begin with, let's get some text up on screen quickly. To do this, you need to get your Amber application server running
+
+### Starting up the Web Server
+
+You actually have a functional Rails application already. To see it, you need to start a web server on your development machine. You can do this by running the following in the`blog`directory:
+
+```bash
+$ amber watch
+```
+
+This will compile your source code and start watching for changes to reload. By default Amber accepts requests on port 3000. If we point our favorite web browser at [http://localhost:3000](http://localhost:3000), we should see the Amber Framework welcome page
+
+![](https://github.com/amberframework/online-docs/blob/master/assets/amber-framework-welcome.png?raw=true)If your screen looks like the image above, congratulations! You now have a working Amber application. If you don’t see the page above, try accessing it via [http://127.0.0.1:3000](http://127.0.0.1:3000) and later make sure your OS has defined “localhost” as “127.0.0.1”.
+
+Locally, the application is running in a Crystal process. To stop it, we hit ctrl-c once, just as we would to terminate the program normally.
+
+> **Note:** The **amber watch** command uses [Sentry](https://github.com/samueleaton/sentry) to watch for any changes in your source files, recompiling automatically. If you don't want to use Sentry, you can compile and run manually:  
+> 1. Build the app `crystal build --release src/[your_app].cr`  
+> 2. Run with `./[your_app]`  
+> 3. Visit `http://0.0.0.0:3000/`
```

Comment:
Also, `amber run` can be used, correct?

Reply:
I think `--release` flag isn't needed on development, it slowdown compilation phase.

`amber run` is ok, but actually it generates the binary on `bin/[your_app]`

Another alternative is using `shards build [your_app]` and runing  `./bin/[your_app]`

Reply:
`amber run` has been removed :sweat_smile: